gcs/deleteFile/deleteFile.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// DeleteFile is a probe that deletes a file from GCS.
+// It tests the file deletion aspect of GCS and ensures that the file
+// is deleted after the deletion request is made.
+//
+// TODOs:
+// TODO(evanSpendlove): Setup labels and metrics within probe
+// TODO(evanSpendlove): Add handling of additional labels
+// TODO(evanSpendlove): Add Start() method to Probe.
+// TODO(evanSpendlove): Add initProbeMetrics() method to Probe.
+// TODO(evanSpendlove): Add runProbeForTarget() method to Probe.
+// TODO(evanSpendlove): Add runProbe() method to Probe.
+
+// Package gcs implements the probes required to monitor a GCS instance.
+package gcs
+
+import (
+	"fmt"
+
+	"github.com/google/cloudprober/logger"
+	"github.com/google/cloudprober/metrics"
+	"github.com/google/cloudprober/probes/options"
+
+	deletepb "github.com/googleinterns/step224-2020/gcs/deleteFile/proto"
+)
+
+// Probe holds aggregate information about all probe runs, per-target.
+// It also holds the config and options used to initialise the probe.
+type Probe struct {
```

Comment:
How about calling this DeleteFileProbe?

Reply:
I wasn't sure if you could rename the Probe struct, but I looked into it a bit further and it looks like you can without breaking Cloudprober compatibility.

Reply:
I've renamed this to `DeleteFileProbe`.